.github/workflows/bump-versions.yaml

```diff
@@ -120,8 +129,12 @@ jobs:
                 patch="0"
                 minor="0"
               fi
-              printf -v version '%d.%d.%d' "$((major))" "$((minor))" "$((patch))"
-            else echo "${version} did not match regex pattern for 1.2.3 or r123"
+              if $startsWithV; then
```

Comment:
new logic here would be 
if $startsWithV: 
   prepend v to version
printf -v version '%d.%d.%d' "$((major))" "$((minor))" "$((patch))"

Reply:
I think my last commit gets at this.

1. Don't use `newVersion` var (move up echo statement for original `version` var so it logs before me mutate it
2. Prepend the `v` in a separate step, so the bulk of the major, minor ,patch formatting is not duplicated.